README.md

```diff
@@ -14,6 +14,26 @@
 
 Download pre-compiled binaries for Mac, Windows and Linux from the [releases](https://github.com/dropbox/dbxcli/releases) page.
 
+### Mac OSX Installation of pre-compiled binaries
+1. If you don't already have it, create a `bin` directory under your home directory.
```

Comment:
I'm all for improving docs, this just feels too specific. How about something simpler like "Make sure you download and place the binary in a folder that's on your $PATH"?

Reply:
Agreed.  The purpose of making it so specific is so we can keep shepherd non-expert users to follow best practices.  Maybe we can add the phrase you suggested on top and say something like:
> Make sure you download and place the binary in a folder that's on your `$PATH`.   If you don't know what that means go to Step 1, otherwise just skip to Step 2.

Maybe something like that?